app/design/adminhtml/default/default/template/sales/order/view/info.phtml

```diff
@@ -63,7 +63,7 @@ $orderStoreDate = $this->formatDate($_order->getCreatedAtStoreDate(), 'medium',
             </tr>
             <tr>
                 <td class="label"><label><?php echo Mage::helper('sales')->__('Purchased From') ?></label></td>
-                <td class="value"><strong><?php echo $this->getOrderStoreName() ?></strong></td>
+                <td class="value"><strong><?php echo $this->getOrderStoreName(); ?></strong></td>
```

Comment:
I think there the `$this->escapeHtml()` call is missing… What do you think @colinmollenhour?

Reply:
I also reported this to Magento, but I didn't receive an answer on this yet.
I'm not sure how the Store Name is saved inside the order - maybe it's sanitized before saving

Reply:
will already sanitize on store saving:

_app/code/core/Mage/Adminhtml/controllers/System/StoreController.php:175_
```php
$postData['website']['name'] = $this->_getHelper()->removeTags($postData['website']['name']);
```

_app/code/core/Mage/Core/Helper/Abstract.php:236_
```php
    public function removeTags($html)
    {
        $html = preg_replace_callback(
            "# <(?![/a-z]) | (?<=\s)>(?![a-z]) #xi",
            function ($matches) {
                return htmlentities($matches[0]);
            },
            $html
        );
        $html =  strip_tags($html);
        return htmlspecialchars_decode($html);
    }['name']);
```

Reply:
removeTags is not escapeHtml though so I don't see why it shouldn't be both as long as there is no double-escaping.